cmd/rpc/Cargo.toml

```diff
@@ -21,3 +21,6 @@ libc = "0.2"
 log = {version = "0.4.8", features = ["std"]}
 parse_int = "0.4.0"
 zerocopy = "0.6.1"
+
+[target.'cfg(windows)'.dependencies]
```

Comment:
Nit: do we want to make the `libc` dependency only included on `not(windows)`?

Reply:
Seems fine to me, I was being lazy; it's probably transitively included anyway.